src/utilities/utilities.hpp

```diff
@@ -206,8 +227,16 @@ struct Arguments {
   bool print_help = false;
   bool silent = false;
   bool no_abbrv = false;
+
+  int tStrategy = DEFAULT_STRATEGY;
+  size_t psoSwarmSize = DEFAULT_PSO_SWARM;
+  double psoInfG = DEFAULT_PSO_G;
+  double psoInfL = DEFAULT_PSO_L;
+  double psoInfR = DEFAULT_PSO_R;
```

Comment:
See above, would also be nice to have these in the appropriate section with the designated comment. Or somehow just within the tuner itself if it's possible to do that nicely.